src/cls/IPM/Storage/Module.cls

```diff
@@ -550,7 +550,19 @@ ClassMethod HasScope(pPhases As %List, pScope As %String) [ Private ]
 		Quit 1
 	}
 	Set tPhases = $zcvt(pPhases,"L")
-	Quit $ListFind(tPhases,pScope)
+	// If pScope contains a comma, treat it as a comma-separated list
+    If $Find(pScope, ",") {
```

Comment:
Would it not ultimately be more readable / cleaner code to keep this as a $ListBuild list, check $ListValid, and iterate over the list with $ListNext rather than $Length/$Piece?